macro/run_simScript.py

```diff
@@ -650,6 +650,38 @@
  rc1 = os.system("rm  "+outFile)
  rc2 = os.system("mv "+tmpFile+" "+outFile)
  fin.SetWritable(False) # bpyass flush error
+
+if simEngine == "muonDIS":
+
+    temp_filename = outFile.replace(".root", "_tmp.root")
+
+    with (
+        ROOT.TFile.Open(outFile, "read") as f_outputfile,
+        ROOT.TFile.Open(inputFile, "read") as f_muonfile,
+        ROOT.TFile.Open(temp_filename, "recreate") as f_temp,
+    ):
+        output_tree = f_outputfile.cbmsim
```

Comment:
Because of #644, it might be better to use `TFile::Get()` until we have switched to 6.32+ only.